src/sherlock/.bower.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
ignore this file and everything int he sherlock folder. remove this from git as well.